src/fides/api/schemas/privacy_notice.py

```diff
@@ -105,6 +107,7 @@ class PrivacyNoticeCreation(PrivacyNotice):
     """
 
     name: str
+    notice_key: FidesKey
```

Comment:
This makes this field required on create.  Do we have the ability to create privacy notices in the frontend yet (I don't think so). If not, I wouldn't consider this being blocked by any frontend change.